### DISCUSSION TITLE
Me indiquem o que estudar pra criar um editor como esse

### DISCUSSION THREAD

**Discussion by CarolFonsecaC** [frontendbr/forum#1469] [Discussão]
Estou me aprofundando em Java Script e PHP e quero iniciar o projeto de um editor, como nesses links:
https://desygner.com/editor/
https://crello.com/pt/artboard/
É possível criar algo assim apenas com JavaScript e PHP, ou terei de estudar outras linguagens? Quais?

**Comment by felipefialho** [comment]
Olá @CarolFonsecaC, acredito que só com JavaScript é possível ter um ótimo resultado.

Olha [um exemplo aqui](https://github.com/artf/grapesjs)

**Comment by CarolFonsecaC** [comment]
Olá @felipefialho , muito obrigada pela dica. Vou aprofundar meus estudos e logo consigo concluir a ideia. 
:D

**Comment by Juuhz** [comment]
Além do Javascript, da uma lida nos conceitos de UX, irá precisar para esse projeto.

**Comment by salatielsql** [comment]
@CarolFonsecaC é bem provavel que eles usam canvas para desenhar então da uma olhada: https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Drawing_shapes
